app/config/ApiDocumentationController.scala

```diff
@@ -34,11 +34,6 @@ class ApiDocumentationController @Inject() (
 ) extends DocumentationController(cc, assets, errorHandler) {
 
   override def definition(): Action[AnyContent] = Action.async {
-    lazy val apiAccess: JsObject = Json.obj(
-      "type"                      -> apiConfig.accessType(),
```

Comment:
Do we not need to keep the "type"? Does apiAccess not get used?

Reply:
we need type as otherwise there is no way to tell the framework that this api is private look here https://catalogue.tax.service.gov.uk/service/get-nino-api/config